shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -18,6 +18,16 @@ if(typeof navigator !== 'undefined'){
 	//Addons
 	require('codemirror/addon/fold/foldcode.js');
 	require('codemirror/addon/fold/foldgutter.js');
+	require('codemirror/addon/fold/xml-fold.js');
```

Comment:
Xml folding doesn't seem to be working for me.

In any case, I kind of want to keep code folding limited to \pages at the moment. Can we remove this for now?

Reply:
Code folding is set to only be our page folding, this is a requirement of the auto-close html feature and not enabled. https://codemirror.net/doc/manual.html#addon_closetag

Reply:
Ah ok. In that case, can we move it next to the closetag require line and maybe  just add a small comment?

Reply:
Absolutely!

Reply:
Comments added and lines re-ordered